### DISCUSSION TITLE
Cannot find root element to mount the component

### DISCUSSION THREAD

**Discussion by timmyExtend** [cypress-io/cypress#20982] [Component Testing] [answered]
Hi Team,

We're starting to introduce Component Testing within our QA lifecycle and are running into an issue where we can't mount the component to the root element. Does anyone have any ideas on what could cause this to happen? 

**The scenario:**
We managed to get the cypress component runner up and running in our monorepo, but when we tried to run our tests we encountered the error listed below. 
Our repo currently consist of the following:
- monorepo
- custom webpack config (using babel)

**The problem:**
<img width="295" alt="Screen Shot 2022-04-07 at 3 00 15 PM" src="https://user-images.githubusercontent.com/67602445/162326876-1f817459-d501-44a3-889d-a5d7d54ae2dd.png">

**Comment by muratkeremozcan** [comment, marked as answer]
@timmyExtend at your company's Client monorepo, check out the branch called `test/NOJIRA-comp-test-poc`
